app/components/SignUp.jsx

```diff
@@ -0,0 +1,119 @@
+import React, {Component} from 'react';
+import TextField from 'material-ui/TextField';
+import RaisedButton from 'material-ui/RaisedButton';
+import {signUp} from 'APP/app/reducers/auth'
+import {connect} from 'react-redux'
+
+const style = {
+  button : {
+    margin: 20
+  },
+  title : {
+    fontFamily : "Roboto, sans-serif",
+    color : "#FA8072"
+  },
+  container : {
+    textAlign : "center",
+    paddingTop : "10%"
+  }
+};
+
+
+
+function mapDispatchToProps(dispatch) {
+  return {
+    signUpSubmit: function(name, email, password) {
+      dispatch(signUp(name, email, password));
+    }
+  }
+}
+
+export default connect (null, mapDispatchToProps) ( 
+  class SignUp extends Component{
+    constructor(){
+      super();
+      this.state = {
```

Comment:
It is a little bizarre that you are using redux to map state to props, but you are also explicitly defining state here. You are sort of half-using redux, which is fine, but something you might want to refactor.